shell/client.go

```diff
@@ -198,6 +214,24 @@ func FormatJsonEasyRead(out bytes.Buffer, rawjson []byte) (outs string, err erro
 	return
 }
 
+func (self *MaestroClient) ConfigNetInterface(args []string) (string, error) {
+	type netIfConfig maestroSpecs.NetIfConfigPayload
+	type netIfConfigs []netIfConfig
+
+	var configs = netIfConfigs{
```

Comment:
Not a bad thing, but would configs := netIfConfigs{   be more golang like?

Reply:
The ":=" syntax seems to be required for capturing multiple return values from a single function call, but here we have a local variable declaration and I looked but can't find a recommended or preferred way to declare variables in this situation.  If anything, I would think "var" would be more Golang-like, simply because that's how everything else is defined (func, type, etc.).  Not really sure and could go either way here.